cmd/kubemark/hollow-node.go

```diff
@@ -181,52 +168,84 @@ func run(config *hollowNodeConfig) {
 	if len(config.TenantServers) == 0 {
 		klog.V(3).Infof("TenantServers is not set. Default to single tenant partition and clientConfig setting")
 		config.TenantServers = make([]string, 1)
-		config.TenantServers[0] = clientConfigs.GetConfig().Host
+		config.TenantServers[0] = config.KubeconfigPath
 	}
 	if config.ResourceServer == "" {
 		klog.V(3).Infof("Resource is not set. Default to clientConfig setting")
-		config.ResourceServer = clientConfigs.GetConfig().Host
+		config.ResourceServer = config.KubeconfigPath
 	}
 
 	// initialize the kubeclient manager
 	kubeclientmanager.NewKubeClientManager()
 
+	// TODO: set this from the hollow node env variable
+	enableInSecurePorts := false
+
 	numberTenantPartitions := len(config.TenantServers)
 	clients := make([]clientset.Interface, numberTenantPartitions)
 	for i := 0; i < len(clients); i++ {
```

Comment:
nit: len(clients) can be numberTenantPartitions directly

Reply:
will make fix. thanks.

Reply:
fixed